stages/org.osbuild.ovf

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python3
+"""
+Create OVF descriptor and manifest
+
+Generates a OVF descriptor (xml) for a vmdk inteded for vSphere.
+
+The OVF descriptor has minimal virtual hardware and no network.
+Hardware and network can be configured during or after importing
+into vSphere.
+
+Buildhost commands used: `qemu-img`.
+"""
+
+import json
+import os
+import subprocess
+import sys
+
+import osbuild.api
+from osbuild.util import checksum
+
+SCHEMA_2 = """
+"options": {
+  "additionalProperties": false,
+  "required": ["vmdk"],
+  "properties": {
+    "vmdk": {
+      "type": "string"
+    }
+  }
+}
```

Comment:
This is the vmdk filename (or path)?  This should be an input to the stage and I guess the stage should have no options (for now).  So the schema would look like:
```json
"options": {
  "additionalProperties": false,
},
"inputs": {
  "type": "object",
  "additionalProperties": false,
  "required": ["vmdk"],
  "properties": {
    "vmdk": {
      "type": "object",
      "additionalProperties": true
    }
  }
}
```

The stage then would then be:
```json
{
  "type": "org.osbuild.ovf",
  "inputs": {
    "vmdk": {
      "type": "org.osbuild.files",
      "origin": "org.osbuild.pipeline",
      "references": {
        "name:image": {
          "file": "image.vmdk"
        }
      }
    }
  }
}
```

(compare with the qemu stage in the test for example).

That way we can reference vmdks from other pipelines if needed, if the ovf stage is executed in its own pipeline for example, so we can export both the vmdk and the ovf.

Reply:
So I wasn't sure. Either we add 2 extra pipelines (vmdk -> ovf (1) -> tar/ova (2)). Or we add only one extra pipeline (tar/ova). I'm open to either, if you think the 2 extra pipelines makes more sense, happy to have that.

Reply:
Let's do 2 and make the `ovf` stage take an input.